tool/latest_flutter_candidate.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright 2022 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+RESPONSE=$(gh api --paginate /repos/flutter/flutter/branches)
```

Comment:
should we delete the latest_flutter_candidate.dart file in this PR too? It is in the repo_tool. We have other scripts that are using it (update_flutter_sdk.sh) that should use this new bash script instead.

Reply:
I was thinking this could be done seperately

Reply:
Looking at it now though and it looks simple enough though. I will add it now :)

Reply:
Feel free to file an issue and we can do in a follow up PR